gui/storage/admin.py

```diff
@@ -344,6 +344,12 @@ def get_actions(self):
         )
 
         # Dataset actions
+        actions['PromoteZFS_dataset'] = self._action_builder(
+            'promote_dataset',
+            label=_('Promote Dataset'),
+            icon="promote_zfs",
```

Comment:
I see no promote_zfs icon in this PR, did you perhaps forgot to commit it?

Reply:
@william-gr Well, yeah...I forgot to add it to the commit